functions/agencies/package.json

```diff
@@ -8,8 +8,7 @@
   "scripts": {
     "test": "tsc && eslint && vitest run --coverage",
```

Comment:
we should probably have a separated command that runs vitest in watch mode, since during development is better to keep constant and quick feedback it also ease TDD development